src/runtime_src/core/common/info_platform.cpp

```diff
@@ -352,6 +352,32 @@ add_clock_info(const xrt_core::device* device, ptree_type& pt)
   }
 }
 
+void
+add_tops_info(const xrt_core::device* device, ptree_type& pt)
+{
+    ptree_type pt_tops_array;
+
+    try {
+        auto res_info = xrt_core::device_query<xq::xrt_resource_raw>(device);
+        if (res_info.empty())
+        return;
+
+        for (auto& res: res_info) {
+            if(res.type == xrt_core::query::xrt_resource_raw::resource_type::ipu_tops_max) {
+                ptree_type pt_tops;
+                pt_tops.add("id", xq::xrt_resource_raw::get_name(res.type));
+                pt_tops.add("value", res.data_double);
+                pt_tops_array.push_back(std::make_pair("", pt_tops));
+                break;
+            }
+        }
```

Comment:
XRT uses early-exit to reduce the level of indentation, please change to:

```suggestion
        for (auto& res: res_info) {
          if (res.type != xrt_core::query::xrt_resource_raw::resource_type::ipu_tops_max) 
            continue;
            
          ptree_type pt_tops;
          pt_tops.add("id", xq::xrt_resource_raw::get_name(res.type));
          pt_tops.add("value", res.data_double);
          pt_tops_array.push_back(std::make_pair("", pt_tops));
          return;
        }
```

Reply:
fixed.